tooling/bin/generate-final-artifact.sh

```diff
@@ -152,6 +152,33 @@ for architecture in "${architectures[@]}"; do
             "${tmp_folder_final_musl_appsec}/etc/recommended.json"
     fi
 
+    ########################
+    # PHP Stubs
+    ########################
+    stubs=(
+        "/home/circleci/datadog/src/ddtrace_php_api.stubs.php"
+        "/home/circleci/datadog/ext/ddtrace.stub.php"
+        "/home/circleci/datadog/ext/hook/uhook.stub.php"
+        "/home/circleci/datadog/ext/hook/uhook_attributes.stub.php"
```

Comment:
Could you please pass `$(PROJECT_ROOT)` in the Makefile and replace /home/circleci/datadog by $3?